startupscript/post-startup.sh

```diff
@@ -73,6 +77,45 @@ ${RUN_AS_LOGIN_USER} "mkdir -p '${USER_BASH_COMPLETION_DIR}'"
 ${RUN_AS_LOGIN_USER} "mkdir -p '${USER_HOME_LOCAL_SHARE}'"
 
 
+#######################################
+# Set guest attributes on GCE. Used here to log completion status of the script.
+# See https://cloud.google.com/compute/docs/metadata/manage-guest-attributes
+# Arguments:
+#   $1: The guest attribute domain and key IE startup_script/status
+#   $2  The data to write to the guest attribute
+#######################################
+function set_guest_attributes() {
+  local attr_path="${1}"
+  local attr_value="${2}"
+  if [ -z "${attr_path}" ] || [ -z "${attr_value}" ]; then
```

Comment:
Added some input checking here.

Reply:
try this one https://github.com/verily-src/workbench-app-devcontainers/blob/6af112ee40d05ee3f1d9188cd4076c65de28e285/startupscript/butane/gcp/metadata-utils.sh#L33C10-L33C22